instrumentation/thrift-0.14.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/thrift/AbstractProtocolWrapper.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.thrift;
+
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.protocol.TProtocolDecorator;
+
+public abstract class AbstractProtocolWrapper extends TProtocolDecorator {
+  public static final String OIJ_THRIFT_FIELD_NAME = "OIJ_FIELD_NAME";
+  public static final short OIJ_THRIFT_FIELD_ID = 3333;
+
+  public AbstractProtocolWrapper(TProtocol protocol) {
```

Comment:
Does this class have any other purpose besides containing `OIJ_THRIFT_FIELD_ID` field?

Reply:
No，just for containing this field